src/main/java/org/apache/sling/servlets/get/impl/DefaultGetServlet.java

```diff
@@ -107,19 +108,19 @@ public class DefaultGetServlet extends SlingSafeMethodsServlet {
         String[] index_files() default { "index","index.html" };
 
         @AttributeDefinition(name = "Enable HTML",
-                description = "Whether the renderer for HTML of the default GET servlet is enabled or not. By default the HTML renderer is enabled.")
+                description = "Whether the renderer for HTML of the default GET servlet is enabled for extension \"html\" or not. By default the HTML renderer is enabled.")
         boolean enable_html() default true;
```

Comment:
Probably we should remove the sentence 

> By default the HTML renderer is enabled.

as the default handling does not really matter (and is to a certain degree visually exposed in the Felix Web Console anyhow)